apps/admin-gui/src/app/admin/pages/admin-page/admin-searcher/admin-searcher.component.ts

```diff
@@ -0,0 +1,164 @@
+import { Component, OnInit } from '@angular/core';
+import {
+  AttributeDefinition,
+  AttributesManagerService,
+  EnrichedFacility,
+  RichMember,
+  RichUser,
+  SearcherService,
+  Vo,
+  VosManagerService,
+} from '@perun-web-apps/perun/openapi';
+import { compareFnDisplayName, compareFnName } from '@perun-web-apps/perun/utils';
+import {
+  TABLE_SEARCHER_FACILITIES,
+  TABLE_SEARCHER_MEMBERS,
+  TABLE_SEARCHER_RESOURCES,
+  TABLE_SEARCHER_USERS,
+} from '@perun-web-apps/config/table-config';
+import { ResourceWithStatus } from '@perun-web-apps/perun/models';
+
+@Component({
+  selector: 'app-admin-searcher',
+  templateUrl: './admin-searcher.component.html',
+  styleUrls: ['./admin-searcher.component.scss'],
+})
+export class AdminSearcherComponent implements OnInit {
+  constructor(
+    private attributesManager: AttributesManagerService,
+    private searcher: SearcherService,
+    private voService: VosManagerService
+  ) {}
+
+  loading: boolean;
+  loadingEntityData = false;
+  allAttrDefinitions: AttributeDefinition[] = [];
+  availableAttrDefs: AttributeDefinition[] = [];
+  searchInput: { [p: string]: string };
+  tableId = TABLE_SEARCHER_USERS;
+  entities: RichUser[] | RichMember[] | EnrichedFacility[] | ResourceWithStatus[] = [];
+  vos: Vo[] = [];
+  selectedVo: Vo;
+
+  ngOnInit(): void {
+    this.loading = true;
+    this.attributesManager.getAllAttributeDefinitions().subscribe((attrDefs) => {
+      this.allAttrDefinitions = attrDefs;
+      this.filterAttrDefinitions('user');
+      this.loading = false;
+    });
+  }
+
+  filterAttrDefinitions(entity: string): void {
+    this.availableAttrDefs = this.allAttrDefinitions
+      .filter((attrDef) => attrDef.entity === entity)
+      .sort(compareFnDisplayName);
+  }
+
+  tabChanged(indexChangeEvent: number): void {
+    this.entities = [];
+    switch (indexChangeEvent) {
+      case 0: // Users
+        this.filterAttrDefinitions('user');
+        this.tableId = TABLE_SEARCHER_USERS;
+        break;
+      case 1: // Members
+        this.loadingEntityData = true;
+        this.filterAttrDefinitions('user'); // we search members in searcher by user attributes !!!
```

Comment:
and what about member attributes?

Reply:
The old GUI is implemented in the same way and also javadocs on the backend mention, that members should be searched by user attributes in the Searcher. Moreover, the name of the function is `getMembersByUserAttributes`, so I think, that it's a feature.